src/message.nim

```diff
@@ -0,0 +1,26 @@
+import std/times
+
+type
+  MessageID* = string
```

Comment:
Wondering if an array or seq of bytes would be more apt for an ID? Same goes for `channelId`